files/galaxy/tpv/destinations.yml.j2

```diff
@@ -233,6 +233,32 @@ destinations:
       require:
         - cz-pulsar
 
+  pulsar_cz02_tpv:
+    inherits: pulsar_default
+    runner: pulsar_eu_cz02
+    max_accepted_cores: 32
+    max_accepted_mem: 128
+    min_accepted_gpus: 1
+    max_accepted_gpus: 1
+    env:
+      LC_ALL: C
+      SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      SINGULARITY_TMPDIR: "/storage/praha5-elixir/home/galaxyeu/singularity/tmp"
+      TMPDIR: "$SCRATCHDIR"
+      TMP: "$SCRATCHDIR"
+      TEMP: "$SCRATCHDIR"
+      ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.3.1"
+      NGPUS: "$PBS_NGPUS"
```

Comment:
I did not know that you needed to set this variable. For my future knowledge, what does it do?

Reply:
:D To be honest, I don't think we need it. I just copy-pasted our (yet not clean) configuration for the first test. We need to specify NCPUS this way for a few tools, and at the time of the first tests on our server, I tried many weird configurations. This is just a relict now. I will remove it eventually.